### PR TITLE
Glowing Text Animation

### LINKED ISSUE #78: Glowing Text Animation
### Prerequisites

- [X] I am an Open Source Contributor
- [X] I checked to make sure that this issue has not already been filed

### Description

I would like to add Glowing Text Animation to the website.

### Screenshots

_No response_

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/Beautiify/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.

### PR DESCRIPTION
**Pull request by GhoshAditi**
# Fixes Issue🛠️
Closes #78

# Description👨‍💻 
Added Glowing Text Animation

# Type of change📄
- [X] New feature (non-breaking change which adds functionality)

# Checklist✅ 

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have added demonstration in the form of GIF/video file
- [X] I am an Open Source Contributor

# Screenshots/GIF📷
beautiify Text

### CODE REVIEW

**Comment by GhoshAditi** [comment]
@Rakesh9100 The glowing text animation is added.

File: assets/html_files/text-animations.html [hunk 1]
```diff
@@ -57,7 +57,8 @@ <h1>Typewriter Text Animation</h1>
                 </div>
             </div>
 
-            <div class="box">
+            <div
+             class="box">
```
Comment (Rakesh9100) [inline assets/html_files/text-animations.html:61]: Move this back in a single line.

Also delete the  empty file `Ghlowing-Text-Animation` @GhoshAditi

**Comment by GhoshAditi** [comment]
@Rakesh9100 Please check whether the mentioned issues are solved or not

**Comment by Rakesh9100** [comment]
Do move back line no. 61 to line 60 in `text-animations.html` file @GhoshAditi

**Comment by GhoshAditi** [comment]
@Rakesh9100  Is it okay now?

**Comment by Rakesh9100** [comment]
Merged!! 🎉 @GhoshAditi

**Comment by rishicds** [comment]
nice one @GhoshAditi

**Pull request merged**